app/src/main/java/io/github/keibai/activities/signin/SignInActivity.java

```diff
@@ -43,6 +52,10 @@ protected void onCreate(Bundle savedInstanceState) {
         ActionBar actionBar = getSupportActionBar();
         actionBar.setDisplayHomeAsUpEnabled(true);
 
+        SharedPreferences sharedPref = getSharedPreferences("UserInfo", 0);
+        et = (EditText) findViewById(R.id.edit_sign_in_email);
```

Comment:
We've got a couple `EditText`. A clearer name would for the variable would probably be better (i.e. editTextEmail).